File: Starting_with_twinder.py
def main():
    from sys import stdin,stdout
    
    m = int(stdin.readline().strip())     
    n = int(stdin.readline().strip()) 
    while n>0:
        h,l = list(map(int, stdin.readline().strip().split(' '))) 
        if h==l and h>=m: stdout.write('ACCEPTED\n')
        elif h>=m and l>=m: stdout.write('CROP IT\n')
        elif h<m or l<m: stdout.write('UPLOAD ANOTHER\n')
        n-=1

if __name__=='__main__':
    main()   

File: java_problems_competitive/StartingWithTwinder.java
package com.test;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class StartingWithTwinder {
	static void print(String s) {
		System.out.println(s);
	}
	
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int m = Integer.parseInt(br.readLine().trim());
		int n = Integer.parseInt(br.readLine().trim());
		
		while(n-->0) {
			String[] hl = br.readLine().trim().split(" ");
			int h = Integer.parseInt(hl[0]);
			int l = Integer.parseInt(hl[1]);
			if (h==l && h>=m) print("ACCEPTED");
			else if (h>=m && l>=m) print("CROP IT");
			else print("UPLOAD ANOTHER");
		}	
	}	
}
